crates/crypto/src/lib.rs

```diff
@@ -289,12 +307,12 @@ pub fn seal_decrypt(
     }
 }
 
-/// Create a full id from the [DST], a package id and an inner id. The result has the following format:
+/// Create a full id from the [DST_ID], a package id and an inner id. The result has the following format:
 /// [len(DST)][DST][package_id][id]
 pub fn create_full_id(package_id: &[u8; 32], id: &[u8]) -> Vec<u8> {
```

Comment:
hmm, perhaps it's better to not add DST_ID here and do that internall in ibe (in hash_to_g1)? it's a bit strange that the domain separator for h1 is defined here implictly

Reply:
Sure, makes sense.